chartfx-chart/src/main/java/de/gsi/chart/axes/spi/AbstractAxis.java

```diff
@@ -152,6 +153,22 @@ public AbstractAxis() {
         widthProperty().addListener(axisSizeChangeListener);
         heightProperty().addListener(axisSizeChangeListener);
 
+        // set default axis title/label alignment
+        sideProperty().addListener((ch, o, n) -> {
+            switch (n) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://app.codacy.com/app/GSI/chart-fx/pullRequest?prid=4700514)